components/modal/types/index.d.ts

```diff
@@ -4,7 +4,7 @@ import { LayerBackdropClickHandler } from '@dhis2-ui/layer'
 export type ModalOnCloseEventHandler = LayerBackdropClickHandler
 
 export interface ModalProps {
-    children?: React.ReactNode
+    children?: React.ReactNode || React.ReactNode[]
```

Comment:
I don't think this is needed? `ReactNode` already allows arrays, through `ReactFragment`-type?

Reply:
Ah yeah, good catch! Plus I put it on the wrong type, should've been ModalTitleProps.